bin/update-patterns

```diff
@@ -9,6 +9,24 @@ function finish {
     docker rm browser-pattern-library &> /dev/null || true
 }
 
+if [[ $# -eq 0 ]]
```

Comment:
I'd pass this script through `shellcheck`, which will probably make you add a lot of `"` around variables for safety and other suggestions

Reply:
Already done!